uint8 foot_id
uint8 LEFT=0
uint8 RIGHT=1
uint8 state
uint8 SUPPORT_LEG=0
uint8 NOT_SUPPORT_LEG=1
geometry_msgs/Polygon polygon
